core/domain/feedback_services.py

```diff
@@ -804,17 +805,18 @@ def _send_feedback_thread_status_change_emails(
                 old_status, new_status)
 
 
-def _check_unique_ids_for_recipients(user_ids, exploration_id, thread_id):
-    user_models = (
-        email_models.FeedbackEmailReplyToIDModel.get_for_multi_user_ids(
+def _check_reply_to_ids_for_recipients(user_ids, exploration_id, thread_id):
```

Comment:
- Add docstring for this function.
- Maybe rename to _ensure_each_recipient_has_reply_to_id? Seems clearer.